src/mixins/reqFulfillment.js

```diff
@@ -51,13 +51,48 @@ export default {
       reqString = reqString.replace(/''/g, '"').replace(/,[\s]+/g, ',');
       const splitReq = reqString.split(/(,|\(|\)|\/)/);
       const _this = this;
+      let skipNumber;
       for (let i = 0; i < splitReq.length; i++) {
-        if (splitReq[i].indexOf('"') >= 0) {
-          // Set strings (like "permission of instructor") automatically to false
+        if (splitReq[i].indexOf('"') >= 0 && i !== skipNumber) {
+          // if the requirement is a string instead of a class ID:
+          // If the string is "One subject in X", check for any subject with X in their ID
+          // things that are still not handled correctly:
+          // 21M.283 doesn't work correctly because film / music
+          // 24.280, 21L.709, 21L.715, 21L.S96, 21L.S97 don't work because we don't keep the count between i's
+          // Set other strings (like "permission of instructor") automatically to false
           // If required as an alternative to other prereqs, will not affect fulfillment
           // If the string is a required prereq, it must be manually dismissed
-          splitReq[i] = 'false';
-        } else if ('()/, '.indexOf(splitReq[i]) < 0) {
+          const req = splitReq[i];
+          let idCategory;
+          let numRequired;
+          if (req.toLowerCase().indexOf('one subject in') >= 0) {
+            if (req.indexOf('ne subject in') >= 0) {
```

Comment:
It would be clearer if this one was also changed to toLowerCase

Reply:
Good point, and also did I delete the check for two subjects in? I should put that back in.

Reply:
Unresolved

Reply:
Which part? I changed it to 'one subject in' last week.

Reply:
I don't see the check for "two subjects in"

Reply:
if (lowercaseReq.indexOf('one subject in') >= 0 || lowercaseReq.indexOf('two subjects in') >= 0) {

Reply:
Ah, must have missed that